seed/views/v3/users.py

```diff
@@ -0,0 +1,883 @@
+                type: string
+                description: the user show shared buildings attribute
+                required: true
+            message:
+                type: string
+                description: error message, if any
+                required: false
+        """
+        ok, content = self.validate_request_user(pk, request)
+        if ok:
+            user = content
+        else:
+            return content
+
+        return JsonResponse({
+            'status': 'success',
+            'show_shared_buildings': user.show_shared_buildings,
+        })
+
+    @ajax_request_class
+    @action(detail=True, methods=['PUT'])
+    def default_organization(self, request, pk=None):
+        """
+        Sets the user's default organization
+        ---
+        parameter_strategy: replace
+        parameters:
+            - name: pk
+              description: User ID (primary key)
+              type: integer
+              required: true
+              paramType: path
+            - name: organization_id
+              description: The new default organization ID to use for this user
+              type: integer
+              required: true
+        type:
+            status:
+                type: string
+                description: success or error
+                required: true
+            message:
+                type: string
+                description: error message, if any
+                required: false
+        """
+        ok, content = self.validate_request_user(pk, request)
+        if ok:
+            user = content
+        else:
+            return content
+        user.default_organization_id = request.query_params.get('organization_id', None)
+        user.save()
+        return {'status': 'success'}
+
+    @ajax_request_class
+    @action(detail=True, methods=['PUT'])
+    def deactivate(self, request, pk=None):
+        """
+        Deactivates a user
+        ---
+        parameter_strategy: replace
+        parameters:
+            - name:
+              description: User ID (primary key)
+              type: integer
+              required: true
+              paramType: path
+            - name: organization_id
+              description: The new default organization ID to use for this user
+              type: integer
+              required: true
+        type:
+            status:
+                type: string
+                description: success or error
+                required: true
+            message:
+                type: string
+                description: error message, if any
+                required: false
+        """
+        user_id = pk
+        # check if user exists
+        user = SEEDUser.objects.filter(
+            id=user_id
+        )
+        if not user.exists():
+            return JsonResponse({
+                'status': 'error',
+                'message': 'user does not exist',
+            }, status=status.HTTP_403_FORBIDDEN)
+
+        user[0].deactivate_user()
```

Comment:
You could also use the try-except pattern with a `SEEDUser.objects.get()` vs the `.filter()`. I'd imagine we'd account for 2 exceptions: 
- [DoesNotExist](https://docs.djangoproject.com/en/2.2/ref/models/instances/#doesnotexist)
- everything else (`except Exception as e:`)

In each case, returning a 400-level response - like you have already for when the user does not exist and like [this](https://github.com/SEED-platform/seed/blob/develop/seed/views/column_mapping_presets.py#L44) for everything else.